resources/js/components/data-list/Table.vue

```diff
@@ -193,7 +193,7 @@ export default {
         },
 
         actualIndex(row) {
-            return _.findIndex(this.sharedState.originalRows, row);
+            return _.findIndex(this.sharedState.originalRows, (originalRow) => originalRow.id === row.id);
```

Comment:
This one was necessary, since after sorting on a column (ex. title) would not be able to find the actual index correctly. It only worked on the initial render.